tests/unit/DataBrowser.spec.js

```diff
@@ -69,11 +69,6 @@ describe('DataBrowser', () => {
           _modelType: 'user',
           _id: 'foo_user_id',
         },
-        selectEnabled: false,
-        multiSelectEnabled: false,
-        uploadEnabled: false,
-        newItemEnabled: false,
-        newFolderEnabled: false,
```

Comment:
I don't think it's a good idea to remove these completely.  Prefer explicitly setting these in the tests rather than relying on the default values.

Reply:
Hm, I'll add them back to make it more explicit.